Doc/reference/datamodel.rst

```diff
@@ -182,6 +182,24 @@ Ellipsis
    related to mathematical numbers, but subject to the limitations of numerical
    representation in computers.
 
+   The string representations of the Numeric classes, computed by
```

Comment:
Ah, this should have been changed to `numeric` too, but that’s very minor so I will leave it and merge the backport PRs as is.